test/utils.js

```diff
@@ -1,6 +1,13 @@
 const crypto = require("crypto");
 const { Algorithm } = require("@iov/bcp-types");
-const { ethereumCodec } = require("@iov/ethereum");
+const { ethereumCodec, EthereumConnection } = require("@iov/ethereum");
+
+async function getBalance(address) {
+  const connection = await EthereumConnection.establish("http://localhost:7545");
+  const account = await connection.getAccount({ address });
+  const balance = account ? account.balance[0].quantity : "0";
```

Comment:
I think we should filter by token ticker "ETH" to be prepared when we soon add other tokens to that list